### PR TITLE
Debian: handle alioth's demise

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
-----

Well this is fun. It's been coming for a while but a bunch of upstreams nonetheless have just... vanished, or moved. FWIW, `libhid` is left out because I cannot find a new homepage for it anywhere, not even a Debian packages reference to point at.

```
Hi, 

after more than two years of preperation I am happy to say that
alioth is history now. I finished the archiving of all vcs repos
yesterday - they are now archived [1]. 

SSH logins on alioth are now disabled, if you find that you are missing
something and think it is *really* urgent - get in touch with me and I should
be able to help you. In a few days we will shutdown and delete the vm. 

The redirector was moved to a DSA managed box, updates to the map will still
happen, but only once a week. 

Whats left?

- deleting the vm
- remove alioth from our docs
- update vcs entries in packages
- improve the salsa docs
- deploy the new sso.debian.org backend

Thats it folks

Thanks to all helping hands

especially ganneff and waldi

Alex - former alioth admin
```

### CODE REVIEW

File: Formula/pinfo.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Pinfo < Formula
   desc "User-friendly, console-based viewer for Info documents"
-  homepage "https://alioth.debian.org/projects/pinfo/"
-  url "https://alioth.debian.org/frs/download.php/file/3351/pinfo-0.6.10.tar.bz2"
+  homepage "https://packages.debian.org/sid/pinfo"
+  url "https://distfiles.macports.org/pinfo/pinfo-0.6.10.tar.bz2"
```
Comment (ilovezfs) [inline Formula/pinfo.rb]: maybe https://mirrorservice.org/sites/distfiles.macports.org/pinfo/pinfo-0.6.10.tar.bz2

**Comment by DomT4** [comment]
> There's https://directory.fsf.org/wiki/Libhid, although it looks like just a brief summary with links to the (now dead) alioth page.

That'll do, thanks! Better than nothing, especially since the formula still compiles fine with the existing inline patch.

File: Formula/pinfo.rb (hunk 1 above)
Comment (DomT4) [reply Formula/pinfo.rb]: Done 👍

**Comment by ilovezfs** [comment]
Thanks for this @DomT4!

**Pull request merged**